### DISCUSSION TITLE
Kafka monitoring OTEL_METRICS_RULES value

### DISCUSSION THREAD

**Discussion by azibhassan** [apache/skywalking#11661] [Q&A]
Hi, 

I am trying to setup kafka monitoring. Currently I am using following ENV Values while setting up OAP Server

```
      SW_OTEL_RECEIVER: default
      SW_OTEL_RECEIVER_ENABLED_OTEL_METRICS_RULES: oap,vm,postgresql/*
      SW_ENABLE_UPDATE_UI_TEMPLATE: true
      SW_TELEMETRY: prometheus
      SW_PROMETHEUS_FETCHER: "default"
      SW_EXPORTER: default
      SW_TELEMETRY_PROMETHEUS_SSL_ENABLED: false
      SW_METER_ANALYZER_ACTIVE_FILES: satellite
```

In my experience, if I don't specify METRIC Rule like vm, oap, I dont see the metrics in SW UI. 

Thanks

**Comment by wu-sheng** [comment]
If you are setting rules by yourself, rather than using default value, yes, you need to add rules.
